.github/workflows/test.yml

```diff
@@ -100,12 +104,17 @@ jobs:
             qemu-user-binfmt
 
       - name: Unit testing (for Linux-Arm64)
-        if: ${{ startsWith(runner.os, 'Linux') }}
+        if: ${{ startsWith(runner.os, 'Linux') && (success() || failure()) }}
```

Comment:
According to https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions the alternative to `always()` seems to be `!cancelled()` (which should be equivalent to `success() || failure()` but shorted)

Reply:
Yes, you are right.
Corrected this [Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild/pull/78/commits/d8ebc2fd7bf590b252b1e25ed3b79709d5ca63d9)